fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -373,6 +375,30 @@ public PlanFragment visitPhysicalHashJoin(PhysicalHashJoin<Plan, Plan> hashJoin,
         }
     }
 
+    @Override
+    public PlanFragment visitPhysicalNestedLoopJoin(PhysicalNestedLoopJoin<Plan, Plan> nestedLoopJoin,
+            PlanTranslatorContext context) {
+        // NOTICE: We must visit from right to left, to ensure the last fragment is root fragment
```

Comment:
Should we add a helper method to wrap this logic?  Maybe something like `private List<PlanFragment>` postOrderVisitChildren(PhysicalPlan plan, PlanVisitor visitor, Context context)`.
Could add a todo and refactor this in the next PR.